Formula/q/qt@5.rb

```diff
@@ -64,7 +64,6 @@ class QtAT5 < Formula
     depends_on "nss"
     depends_on "opus"
     depends_on "pulseaudio"
-    depends_on "re2"
```

Comment:
I think this should avoid our `re2` in CI environment. It may still be autodetected if `re2` is already installed. Not sure what flag we need to pass (i.e. is it `-no-webengine-re2`? or is that for disabling feature itself?). Not too important until we need to modify both in same PR.

Anyways, trying to backport Chromium fixes is not going to be worth the effort on `qt@5` which we are trying to reduce usage. Most upstream support is for paying customers and not really keeping dependencies up-to-date (which is why we have various patches that may never go away).

Reply:
This worked for `qt` so I hope we are fine here now. I agree that dropping is probably the easiest solution right now.